specs/latest/2.0/index.html

```diff
@@ -1370,40 +1466,29 @@ <h2 class="no-toc">Table of contents</h2>
+      <dt class="idl-code">
+        void texImage2D(GLenum target, GLint level, GLint internalformat,
+                        GLsizei width, GLsizei height, GLint border,
+                        GLenum format, GLenum type, ByteBuffer? pixels,
+                        optional unsigned long byteOffset = 0)
+        <span class="gl-spec">(<a href="http://www.khronos.org/registry/gles/specs/3.0/es_spec_3.0.4.pdf#nameddest=section-3.8.3">OpenGL ES 3.0.4 &sect;3.8.4</a>, <a class="nonnormative" href="https://www.khronos.org/opengles/sdk/docs/man3/html/glTexImage2D.xhtml">man page</a>)</span>
       </dt>
       <dd>
         <p>Only differences from <a href="../1.0/index.html#TEXIMAGE2D">texImage2D</a> in WebGL 1 are described here. </p>
         <p>If a WebGLBuffer is bound to the <code>PIXEL_UNPACK_BUFFER</code> target, generates an <code>INVALID_OPERATION</code> error.</p>
         <p>Sized internal formats are supported in WebGL 2 and <em>internalformat</em> is no longer required to be the same as <em>format</em>. Instead, the combination of <em>internalformat</em>, <em>format</em>, and <em>type</em> must be listed in Table 1 or 2 from <span class="gl-spec"><a class="nonnormative" href="https://www.khronos.org/opengles/sdk/docs/man3/html/glTexImage2D.xhtml">man page</a></span>.</p>
-        <p>If <em>type</em> is specified as <code>FLOAT_32_UNSIGNED_INT_24_8_REV</code>, <em>pixels</em> must be null; otherwise, generates an <code>INVALID_OPERATION</code> error.</p>
-        <p>If <em>pixels</em> is non-null, the type of <em>pixels</em> must
-match the <em>type</em> according to the following table; otherwise, generates an <code>INVALID_OPERATION</code> error:</p>
-        <table id="TEXTURE_PIXELS_TYPE_TABLE">
-          <tr><th>type of <em>pixels</em></th><th><em>type</em></th></tr>
-          <tr><td>Int8Array</td><td>BYTE</td></tr>
-          <tr><td>Uint8Array</td><td>UNSIGNED_BYTE</td></tr>
-          <tr><td>Int16Array</td><td>SHORT</td></tr>
-          <tr><td>Uint16Array</td><td>UNSIGNED_SHORT</td></tr>
-          <tr><td>Uint16Array</td><td>UNSIGNED_SHORT_5_6_5</td></tr>
-          <tr><td>Uint16Array</td><td>UNSIGNED_SHORT_5_5_5_1</td></tr>
-          <tr><td>Uint16Array</td><td>UNSIGNED_SHORT_4_4_4_4</td></tr>
-          <tr><td>Int32Array</td><td>INT</td></tr>
-          <tr><td>Uint32Array</td><td>UNSIGNED_INT</td></tr>
-          <tr><td>Uint32Array</td><td>UNSIGNED_INT_5_9_9_9_REV</td></tr>
-          <tr><td>Uint32Array</td><td>UNSIGNED_INT_2_10_10_10_REV</td></tr>
-          <tr><td>Uint32Array</td><td>UNSIGNED_INT_10F_11F_11F_REV</td></tr>
-          <tr><td>Uint32Array</td><td>UNSIGNED_INT_24_8</td></tr>
-          <tr><td>Uint16Array</td><td>HALF_FLOAT</td></tr>
-          <tr><td>Float32Array</td><td>FLOAT</td></tr>
-        </table>
```

Comment:
It might not break WebGL 1.0 content, since the old overloads will be picked up from WebGLRenderingContextBase -- but it will break the WebGL 2.0 conformance tests.